### PR TITLE
sdformat4 4.0.0

### PR DESCRIPTION
**Pull request by osrfbuild**

### CODE REVIEW

File: sdformat4.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Sdformat4 < Formula
   desc "Simulation Description Format"
   homepage "http://sdformat.org"
-  url "http://gazebosim.org/distributions/sdformat/releases/sdformat-4.0.0~pre3.tar.bz2"
-  version "4.0.0~pre3"
-  sha256 "944a187920603929e68bc31fbce6829dcb1097c0ccaa7e9922306384fcb50644"
+  url "http://gazebosim.org/distributions/sdformat/releases/sdformat-4.0.0.tar.bz2"
+  version "4.0.0"
```
Comment (scpeters) [inline sdformat4.rb]: when we switch from prerelease to an official release, we can delete the `version` tag. I'll do this manually

Comment (scpeters) [inline sdformat4.rb]: 2047c51

**Comment by scpeters** [comment]
@j-rivero I'd like to merge this and then add bottles separately. How does that sound?

**Comment by j-rivero** [comment]
> @j-rivero I'd like to merge this and then add bottles separately. How does that sound?

ok for me, the status of this PR probably does not have too much to do with the rest.

**Pull request closed without merging**